### ISSUE TITLE
Unable to install to Python 3.6

### ISSUE THREAD

**Issue by Alienater** [hplgit/odespy#11]
If someone with write access like @rothnic could update this to work with the latest Python version I would greatly appreciate it. I am new to Python and could use odespy in a class I'm taking. 

Thank you

**Comment by renatocoutinho** [comment]
This issue has been open for over 2 years, and I just re-fixed it without realizing there was an open PR already. I think the main issue is that the project needs a new maintainer. For that, the original creators need at least to settle on a license, otherwise we can't just pick up the code and move on.

**Comment by WenyinWei** [comment]
Yeah, it is time to move on to the python3 version. Anticipate this package's improvement.

**Comment by ChaosCom** [comment]
Just a short fyi: According to the "Programming for Computations" book, the 3.6+ version is located at the following repository: https://github.com/thomasantony/odespy/tree/py36 (note the "py36" branch)
